specification/mediaservices/resource-manager/Microsoft.Media/stable/2020-05-01/examples/liveevent-update.json

```diff
@@ -54,7 +51,7 @@
           "provisioningState": "InProgress",
           "created": "0001-01-01T00:00:00Z",
           "lastModified": "0001-01-01T00:00:00Z",
-          "vanityUrl": false,
+          "useStaticHostname": false,
```

Comment:
any particular reason to change the property name? This would cause SDK breaking change across versions.